tests/vitest/skins.citizen.scripts/overflowElements.test.js

```
@@ -0,0 +1,391 @@
+}
+
+/**
+ * Create a bodyContent element attached to the document.
+ * Elements must be in the DOM so parentNode is available for the wrapper module.
+ *
+ * @param {HTMLElement[]} children
+ * @return {HTMLElement}
+ */
+function createBodyContent( children = [] ) {
+	const bodyContent = document.createElement( 'div' );
+	children.forEach( ( child ) => bodyContent.appendChild( child ) );
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)